package.json

```diff
@@ -119,7 +119,7 @@
     "react-native-confirmation-code-field": "5.1.0",
     "react-native-default-preference": "1.4.1",
     "react-native-device-info": "5.5.6",
-    "react-native-elements": "0.19.0",
+    "react-native-elements": "^2.1.0",
```

Comment:
if you upgraded the react-native-elements library please check if you can remove this nested package.json with the node modules that it installed
![Zrzut ekranu 2020-08-3 o 08 28 00](https://user-images.githubusercontent.com/67273035/89152369-51424e80-d563-11ea-8fd5-3f513c178359.png)